.azure/pipelines/ci.yml

```diff
@@ -53,6 +94,18 @@ jobs:
     - script: "echo ##vso[build.addbuildtag]release-candidate"
       condition: and(ne(variables['Build.Reason'], 'PullRequest'), in(variables['DotNetFinalVersionKind'], 'release', 'prerelease'))
       displayName: 'Set CI tags'
+    - task: DownloadBuildArtifacts@0
```

Comment:
This is a lot of repetition. There may be a way for templates to inherit from each other. Will look into it.